### ISSUE TITLE
Not displaying imgs

### ISSUE THREAD

**Issue by Hazarzir** [KeatonTech/Angular-2-Page-Slider#15]
Hey! Super nice work here!!
I've used it in my project... Problem is- the component appears in the DOM, but first, it has a css property- **overflow: hidden**. Only once I cancel it the component appears in the DOM. 
But the bigger problem for me is that the imgs are not there in the DOM... First I used the default paths of the "hero" example. Didn't work.

Only after I created a new folder under my Angular's "assets" folders, named it "img" and added 2 png files there. I changed the paths in the pages array:
    public pages = [
    {image: '../../assets/img/pepperoni_pizza.jpg'},
    {image: '../../assets/img/sushi.jpg'}
  ];

Still... not workong... Am I missing something out?

TNX!!!!!!

**Comment by KeatonTech** [comment]
Hmm, I'm not sure I understand the problem well enough to help out. Can you put together a jsfiddle or something like that?

Generally, `overflow: hidden` is necessary to keep elements from one page from jutting out into another. Are you using CSS floats or absolute positioning or something like that? Those may cause the page to measure its height as 0px, which would then cause everything to be hidden.

I can't speak to your image loading issue. Do you think that's related to this component or could it be something about how you're serving or compiling the app?

**Comment by nukec** [comment]
I have the same issue. I installed and followed rules, i can see empty, when i disable overflow:hidden i can see buttons and use prev next

**Comment by ghost** [comment]
+1

**Comment by nukec** [comment]
since this is outdated and not maintained anymore, i just created my stuff in angular, it's very simple and straight forward.

**Comment by ghost** [comment]
Yeah i know its simple, but i needed this for a quick display project. If someone else has reached this page with similar needs. I would suggest using [angular2-useful-swiper](https://www.npmjs.com/package/angular2-useful-swiper). Its pretty straight forward and easy to use.
